packages/themeables/background/meta.tsx

```diff
@@ -38,34 +41,38 @@ const newTheme: TThemeableBackgrounds<Mappings> = {
   }),
 }
 
-type TDemoComponent = TDemo & { hasTheme: boolean}
+type TDemoComponent = {
+  hasTheme: boolean,
+  withOverride: boolean,
+} & TDemo
+
+const overrideTheme: TThemeables<TThemeableBackground, Mappings> = {
+  demo: ({ status }) => ({
+    color: status === 'error' ? [144, 24, 56, 1] : [108, 105, 86, 1],
+  }),
+}
 
-export const Component = ({ status, hasTheme }: TDemoComponent) => (
-  <Block
-    style={{
-      width: 100,
-      height: 100,
-    }}
-  >
-    {(
-      hasTheme
-        ? (
-          <BackgroundTheme.Provider value={newTheme}>
-            <DemoThemeableBackground status={status}/>
-          </BackgroundTheme.Provider>
-        )
-        : (
-          <DemoThemeableBackground status={status}/>
-        )
-    )}
-  </Block>
+export const Component = ({ status, hasTheme, withOverride }: TDemoComponent) => (
+  <OverrideContext.Provider value={withOverride ? overrideTheme : {}}>
+    <Block
+      style={{
+        width: 100,
+        height: 100,
+      }}
+    >
+      <BackgroundTheme.Provider value={hasTheme ? newTheme : defaultTheme}>
+        <DemoThemeableBackground status={status}/>
+      </BackgroundTheme.Provider>
+    </Block>
+  </OverrideContext.Provider>
 )
 
 Component.displayName = 'ThemeableBackground'
 
 export const config: TComponentConfig<TDemoComponent> = {
   props: {
     hasTheme: [true],
+    withOverride: [true],
```

Comment:
not sure I follow the logic here, why not using something like: 
```
overrideTheme: [overrideTheme, {}]
```
for better readability?

Reply:
I'm not sure what you're referring to 🤔 This line is what sandbox expects. Based on that it will render a `withOverride` that will be a switch (so it only takes true | false).

Reply:
yea, no what i was thinking that in bubble-next we try to make the props here more like in the component (the component does not have `withOverride` prop, but the sandBox example does).
but i guess this approach is more readable in sandbox 👍